packages/activation-notice/src/app/components/step2.js

```diff
@@ -18,14 +18,13 @@
  * External dependencies
  */
 import styled from 'styled-components';
-import { TranslateWithMarkup } from '@web-stories-wp/i18n';
 import { trackClick } from '@web-stories-wp/tracking';
 
 /**
  * WordPress dependencies
  */
 import { __, _x } from '@wordpress/i18n';
-import { useCallback } from '@wordpress/element';
+import { createInterpolateElement, useCallback } from '@wordpress/element';
```

Comment:
I think this needs to be mocked in the JS unit tests. They are currently failing.